plugins.d/alarm-notify.sh

```diff
@@ -1339,6 +1363,62 @@ send_telegram() {
 }
 
 # -----------------------------------------------------------------------------
+# Microsoft Team sender
+
+send_msteam() {
+
+    local webhook="${1}" channels="${2}" httpcode sent=0 channel color payload
+
+    [ "${SEND_MSTEAM}" != "YES" ] && return 1
+
+    case "${status}" in
+        WARNING)  icon="${MSTEAM_ICON_WARNING}" && color="${MSTEAM_COLOR_WARNING}";;
+        CRITICAL) icon="${MSTEAM_ICON_CRITICAL}" && color="${MSTEAM_COLOR_CRITICAL}";;
+        CLEAR)    icon="${MSTEAM_ICON_CLEAR}" && color="${MSTEAM_COLOR_CLEAR}";;
+        *)        icon="${MSTEAM_ICON_DEFAULT}" && color="${MSTEAM_COLOR_DEFAULT}";;
+    esac
+
+    for channels in ${channels}
```

Comment:
this is a typo... it should be:

```
for channel in ${channels}
```

Reply:
hm... I also don't see the variable `channel` used anywhere...
This means channels are not actually used.
Are you sure this works as expected?

Reply:
I made some changes according to your comment. I'm very disapointed about MS Teams regarding the way to push webhook on it. MS copied Slack but not all the best features ... Let me know if you are OK with my way to implement it. Thanks.